src/pages/[guild]/collect/[chain]/[address].tsx

```diff
@@ -175,13 +182,37 @@ const Page = ({
                   <CollectibleImage src={image} isLoading={isLoading} />
 
                   <Stack spacing={6}>
-                    <Heading
-                      as="h2"
-                      fontFamily="display"
-                      fontSize={{ base: "3xl", lg: "4xl" }}
+                    <HStack
+                      flex="1 0 auto"
```

Comment:
Don't we just need `justifyContent="space-between"` here? 👀

Reply:
Fixed